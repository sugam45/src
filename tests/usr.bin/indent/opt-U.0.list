/* $NetBSD: opt-U.0.list,v 1.1 2021/03/06 17:56:33 rillig Exp $ */
/* $FreeBSD$ */

/*
 * From each line of this file, the first word is taken to be a type name.
 *
 * Since neither '/*' nor '' are syntactically valid type names, this means
 * that comments like this one are effectively ignored.  When a type name is
 * indented by whitespace, it is ignored as well.
 *
 * Since only the first word of each line is relevant, any remaining words
 * can be used for comments.
 */

size_t			from stddef.h
off_t			for file offsets
 ignored_t		is ignored since it is indented
