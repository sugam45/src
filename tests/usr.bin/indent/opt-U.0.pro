/* $NetBSD: opt-U.0.pro,v 1.1 2021/03/06 17:56:33 rillig Exp $ */
/* $FreeBSD$ */

-Uopt-U.0.list
