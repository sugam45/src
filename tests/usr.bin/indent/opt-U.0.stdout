/* $NetBSD: opt-U.0.stdout,v 1.2 2021/03/07 10:12:18 rillig Exp $ */
/* $FreeBSD$ */

void
example(void *arg)
{
	int		known_1 = (size_t)*arg;
	int		known_2 = (off_t)*arg;
	int		ignored = (ignored_t) * arg;
}
